Source/Core/DolphinQt/Config/NewPatchDialog.cpp

```diff
@@ -84,6 +85,23 @@ void NewPatchDialog::AddEntry()
   m_entry_layout->addWidget(CreateEntry(m_patch.entries[m_patch.entries.size() - 1]));
 }
 
+void NewPatchDialog::OnTextEdited(QLineEdit* edit, const QString& text, u32* field)
```

Comment:
Apologies, I should have been more precise. This doesn't use `this` at all, so I would turn this into either a static free function (`static void OnTextEdited(...)`) or into a lambda next to the connect calls.